tests/entrypoints/openai/test_load.py

```diff
@@ -0,0 +1,105 @@
+"""
+This file tests significant load on the vLLM server.
```

Comment:
cc @simon-mo --- this test takes ~3 minutes on H100

Will likely take >10 min on L4 ... are you okay with this?